src/quo2/components/navigation/floating_shell_button/style.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo2.components.navigation.floating-shell-button.style
+  (:require [react-native.reanimated :as reanimated]))
+
+(defn floating-shell-button
+  [style opacity-anim]
+  (reanimated/apply-animations-to-style
+   (if opacity-anim
+     {:opacity opacity-anim}
+     {})
+   (merge
+    {:flex-direction     :row
+     :height             44
+     :align-self         :center
+     :padding-top        8
+     :padding-horizontal 12
```

Comment:
Except file rename and break into view and style, only this section changed in floating-shell-button component